examples/demo-rollup/README.md

```diff
@@ -33,11 +33,59 @@ Sync a Celestia light node running on the Arabica testnet
 1. Initialize the node: `celestia light init --p2p.network arabica`
 1. Start the node with rpc enabled. Our default config uses port 11111: `celestia light start --core.ip https://limani.celestia-devops.dev --p2p.network arabica --gateway --rpc.port 11111`. If you want to use a different port, you can adjust the rollup's configuration in rollup_config.toml.
 1. Obtain a JWT for RPC access: `celestia light auth admin --p2p.network arabica`
-1. Copy the JWT and and store it in the `celestia_rpc_auth_token` field of the rollup's config file (`rollup_config.toml`). Be careful to paste the entire JWT - it may wrap across several lines in your terminal.
+1. Copy the JWT and store it in the `celestia_rpc_auth_token` field of the rollup's config file (`rollup_config.toml`). Be careful to paste the entire JWT - it may wrap across several lines in your terminal.
 1. Wait a few minutes for your Celestia node to sync. It needs to have synced to the rollup's configured `start_height `293681` before the demo can run properly.
 
 Once your Celestia node is up and running, simply `cargo +nightly run` to test out the prototype.
 
+### Submitting transactions
+
+You can use either the rest API or celestia-appd. The following instructions assume celestia-appd.
+For testing, we can submit a transaction to the bank module to create a new token
+
+* Ensure demo-rollup is running in one window following the steps from the previous section, and that it's caught up
+
+### Install celestia-appd
+1. Install Go 1.20 - https://go.dev/doc/install
+2. Clone the repository: `git clone https://github.com/celestiaorg/celestia-app.git`.
+3. `cd celestia-node`
+4. Check out tag v0.13.0 - `git checkout tags/v0.13.0`
+5. `make install`
+
+### Create local keypair
+1. `celestia-appd keys add sequencer_keypair` (this will be the sequencer da keypair)
+2. For the arabica testnet, you can get tokens from the arabica-faucet channel in the celestia discord https://discord.gg/celestiacommunity
+
+### Create bank transaction
+1. `cd ../../` (sovereign root)
+2. `cargo build --release --bin bank-cmd`
+3. `./target/release/bank-cmd create-private-key .` - this is the rollup private key that's used to sign rollup transactions. its important to make the distinction between this key and the sequencer private key
+4. `ls -lahtr | grep sov1` - you should see a new json file created containing the keypair
+5. ```./target/release/bank-cmd serialize-call sov1pc06camefctl48ta76rnppdux8r4v5x54yrxred3hhxyqctsv2hshtewjw.json examples/demo-stf/src/bank_cmd/test_data/create_token.json 0 ```
+6. Get the token address from the above the command. eg: `sov1jzvd95rjx7xpcdun2h8kyqee2z5r988h3wy4gsdn6ukc5ae04dvsrad3jj`
```

Comment:
`sov1pc06camefctl48ta76rnppdux8r4v5x54yrxred3hhxyqctsv2hshtewjw.json ` will change every time a new private key is generated. Maybe let's skip point 3 and tell user to use our test private key `token_deployer_private_key`?

Reply:
as discussed, using a newly generated key (for nonce purposes) but changed the doc to reflect that its the generated file (so they don't mistakenly copypaste)